src/client/src/components/Dropdown/dropdownstyles.tsx

```diff
@@ -62,8 +61,8 @@ export default {
     ...base,
     // kill the white space on first and last option
     padding: 0,
-    background: "var(--vscode-editor-background)",
-    border: "0.5px solid var(--vscode-editor-foreground)",
+    background: "var(--vscode-editorWidget-border);",
```

Comment:
is the semicolon necessary here?

Reply:
I think either works - I was trying to be consistent but seems like there's more I missed